doc/breaking-changes-0.8.md

```diff
@@ -0,0 +1,151 @@
+                "createdAt": "2020-01-04T02:34:00.788444359Z",
+                "updatedAt": "2020-01-04T02:34:00.788444359Z",
+                "_links": {
+                    "edge-routers": {
+                        "href": "./services/1012d4d7-3ab3-4722-8fa3-ae9f4da3c8ba/edge-routers"
+                    },
+                    "self": {
+                        "href": "./services/1012d4d7-3ab3-4722-8fa3-ae9f4da3c8ba"
+                    },
+                    "service-policies": {
+                        "href": "./services/1012d4d7-3ab3-4722-8fa3-ae9f4da3c8ba/identities"
+                    }
+                },
+                "tags": {},
+                "name": "cac9593c-0494-4800-9f70-c258ff28a702",
+                "dns": {
+                    "hostname": "0bf71754-ed5b-4b2d-9adf-a542f1284275",
+                    "port": 0
+                },
+                "endpointAddress": "4662d564-3fc3-4f10-b8cd-ee0e3629ad24",
+                "egressRouter": "aedab92f-2ddf-445a-9194-73d428322a34",
+                "edgeRouterRoles": null,
+                "roleAttributes": [
+                    "2c68789a-fe71-4d25-a483-43e54ee4fd98"
+                ],
+                "permissions": [
+                    "Bind"
+                ]
+            }
+        }
+```
+
+### Appwan Removal and Migration
+The `/app-wans` endpoint has been removed. The bbolt schema version has been bumped to 3. If starting a fresh controller no action will be taken. However, if coming from an existing 0.7 or earlier bbolt database, the following will be done:
+
+  1. For each existing appwan, a service policy with type "Dial" will be created 
+  1. The new service policy will have the same name as the appwan it replaces 
+  1. The new service policy will have the same identities and services as the appwan it replaces
+  1. Identities and services will be specified explicitly by ID rather as opposed to by creating new role attributes 
+
+NOTE: Service hosting identities will not be migrated into equivalent Bind service policies, as binds are not yet used in any production scenarios. 
+
+# Go SDK changes
+Several types have been renamed to conform to standard nomenclature
+
+  * Session is now ApiSession
+  * NetworkSession is now Session
+     * The SessionId field is now ApiSessionId
+     * The Gateways field is now EdgeRouters
+  * Gateway is now EdgeRouter
+  * On the Service type the Hostable flag has been removed and replaced with a Permissions string array
+      * It may be nil, empty or contain either or both of "Dial" and "Bind" 
+  * On the Context type
+      * GetNetworkSession is now GetSession
+      * GetNetworkHostSession is now GetBindSession
+      
+# ziti command line changes
+  1. The `ziti edge controller create/delete gateway` commands have been removed. Use `ziti edge controller create/delete edge-router` instead.
+  2. There are new `ziti edge controller create/delete service-policy` commands
```

Comment:
thank you for this - and nicely doc'ed/accounted for!

Reply:
:+1: :)